Core/src/IfcGeometryConverter/RepresentationConverter.h

```diff
@@ -513,9 +513,9 @@ namespace OpenInfraPlatform {
 
 				/*! \brief Converts \c IfcTopologicalRepresentationItem to meshes and polylines.
 				 *
-				 * \param[in] mapped_item The \c IfcTopologicalRepresentationItem to be converted.
-				 * \param[in] pos The relative location of the origin of the representation's coordinate system within the geometric context.
-				 * \param[out] itemData A pointer to be filled with the relevant data.
+				 * \param[in] topo_item			The \c IfcTopologicalRepresentationItem to be converted.
+				 * \param[in] objectPlacement	The relative location of the origin of the representation's coordinate system within the geometric context.
+				 * \param[out] itemData			A pointer to be filled with the relevant data.

```

Comment:
I know "pos" is maybe not the best name to understand what is meant but I'm quite sure that pos is used in a lot of converter functions 
So having multiple names for the same thing might not be the best idea 
But it's just a minor comment and I leave it up to you if you want to change it

Reply:
I see the point of the more understandable name `objectPlacement`.
With a rough search, VS found 241 matches in 6 files (.Core project). In case we want to change this name, a new PR would be suitable in my opinion.
Anyway, I have made a note to not forget this in the next meeting where we could talk more about this?

Reply:
An issue as reminder and basis for a new PR is created.